relentless/simulate/simulate.py

```diff
@@ -391,6 +394,19 @@ def rmax(self):
     def rmax(self, val):
         self.stop = val
 
+    @property
+    def neighbor_buffer(self):
+        """float: The amount to be added to ``rmax`` to search for particles while
+        computing the neighbor list."""
+        return self._neighbor_buffer
+
+    @neighbor_buffer.setter
+    def neighbor_buffer(self, val):
+        if val is not None:
+            self._neighbor_buffer = val
+        else:
+            self._neighbor_buffer = None
```

Comment:
This branch is redundant:
```suggestion
        self._neighbor_buffer = val
```